### DISCUSSION TITLE
Is there a figma design file for this web tool kit?

### DISCUSSION THREAD

**Discussion by techsupport0001** [wet-boew/wet-boew#9609] [Q&A]
What's the best way to get designers to use this tool kit? Is there a design file for XD, Figma or sketch? Not sure where to start. If anyone can help that would be great!

**Comment by ghost** [comment]
Not that I am aware. Main css is Bootstrap 3 with custom css on top for styling. There been some porting of Bootstrap 4/5 classes but no official full support yet.

For more details on Bootstrap stuff I take a read https://github.com/wet-boew/wet-boew/issues/9508#issuecomment-1416427855

###### I am just a volunteer on this GitHub project.

**Comment by AleksueiR** [comment]
Hey, @techsupport0001 , 

I'm part of the UX team at ECCC and we've recently published our [Web Experience Toolkit and Canada.ca Design System Kit](https://www.figma.com/community/file/1342923790850747276/web-experience-toolkit-and-canada-ca-design-system-kit) to the Figma community. This resource is specifically developed to support design of WET and Canada.ca-related projects and we are very excited to share it 🎉. Feel free to check it out and see if it fits your use case:

[![598437bd-1096-437c-ab78-31ad809ff143-cover](https://github.com/wet-boew/wet-boew/assets/2285779/10d050f8-774e-48ff-9b1e-9f5d9a12f932)](https://www.figma.com/community/file/1342923790850747276/web-experience-toolkit-and-canada-ca-design-system-kit)

This is something we developed internally for our own use to speed up the designing process, but thought it might be helpful to other folks out there who design for WET. 

This library is not limited to vanilla WET components, but also includes Canada.ca style guide elements and components from the Centrally Deployed Templates Solution (CDTS) templates.

If you see any missing elements that you need or have other suggestion for improvements, let us know and we'll try to add them.

Thank you!
